### ISSUE TITLE
Windows 11 reset and now it's asking for a username

### ISSUE THREAD

**Issue by hamishakl** [AveYo/MediaCreationTool.bat#364]
Windows has reset and it's asking for a username when I never had one to begin with.

**Comment by harryytran** [comment]
Did you ever find a fix for this ?

**Comment by hamishakl** [comment]
Unfortunately not. Just reimaged and used a different tool. Plenty of others on Github that work without a problem

**Comment by LightYagami28** [comment]
Certainly! Here’s a message you can use to tag hamishalk and inquire about the issue resolution:

Hi @hamishakl,

I noticed that the issue is still unresolved. Could you please let us know if you've managed to resolve it with the alternative tool or if there’s any further assistance needed?

If the issue is resolved, kindly close the issue or let us know so we can proceed accordingly.

Thank you!
